bsp/coreip-e20-arty/metal-inline.h

```diff
@@ -1,7 +1,6 @@
 /* Copyright 2019 SiFive, Inc */
 /* SPDX-License-Identifier: Apache-2.0 */
 /* ----------------------------------- */
-/* [XXXXX] 28-05-2019 10-05-52        */
 /* ----------------------------------- */
```

Comment:
If you rebase on development-19.05 it'll clean up the diff for all of these changes.

Reply:
Yes. I've rebased the develop-19.05 branch. Thanks :)